example/assert_handler.cpp

```diff
@@ -6,13 +6,16 @@
 
 #define BOOST_ENABLE_ASSERT_HANDLER
 
+#include <array>   // std::array
 #include <cstdlib> // std::exit
-#include <boost/array.hpp>
+
+#include <boost/config.hpp>  // BOOST_NOINLINE
+
 BOOST_NOINLINE void foo(int i);
 BOOST_NOINLINE void bar(int i);
  
 BOOST_NOINLINE void bar(int i) {
-    boost::array<int, 5> a = {{101, 100, 123, 23, 32}};
+    std::array<int, 5> a = {{101, 100, 123, 23, 32}};
```

Comment:
Here, and in the following tests, replace `std::array` with a C array `int a[5] `